pkg/telemetry/telemetry.go

```diff
@@ -121,6 +125,10 @@ func (t *engineImpl) writeToTimescaleMetadataTable(m Metadata) {
 	}
 }
 
+func (t *engineImpl) Write(key, value string) {
+	t.writeToTimescaleMetadataTable(map[string]string{key: value})
```

Comment:
Write to DB can fail so I'd recommend returning and checking error

Reply:
There is no error returned for metadata. Plus, writing a telemetry if errors should not be reported ideally.

Reply:
Not sure I follow your logic. Imho any error should be at least logged. If our current api ignores error I think we should change it.